pkg/client/factory.go

```diff
@@ -49,7 +55,19 @@ func NewFactory(baseName string) Factory {
 		flags:    pflag.NewFlagSet("", pflag.ContinueOnError),
 		baseName: baseName,
 	}
+
+	if config, err := LoadConfig(); err == nil {
+		f.namespace = config[ConfigKeyNamespace]
+	} else {
+		fmt.Fprintf(os.Stderr, "WARNING: error retrieving namespace from config file: %v\n", err)
```

Comment:
this will now warn every time if a user hasn't set up a config file, right? i'd say this is probably undesirable since the majority of folks will just continue running with the default.

Reply:
No, see the godoc/impl for LoadConfig

Reply:
OK, so this would only print if the file exists but does not contain namespace. Could be an issue in the future if we add additional fields to client config (so a user has another field set but not namespace), but fine for now.

Reply:
No, it will only print if there is an error opening the file or decoding its contents as json to a map[string]string.

Reply:
d'oh. brain's still on vacation!